dev/extractor.py

```diff
@@ -0,0 +1,176 @@
+#!/bin/env python3
+import os
+import sys
+import subprocess
+import webbrowser
+import time
+import threading
+import shutil
+import tempfile
+import json
+from urllib import request
+
+
+class LMMSIOGettext(object):
+
+    def __init__(self, debug=False, output=None):
+        self.debug = debug
+        self.runner = self.headless_run
+        self.php_srv = None
+        self.output = output
```

Comment:
Some of the files here use spaces rather than tabs for indentation. Do you think we should change that to be consistent or we could just leave it as-is?